app/models/software_description_fetchers/strategies/default.rb

```diff
@@ -29,10 +30,10 @@ def clean_text(text)
       end
 
       def regex
-        /(^.*(?:#{@software_name}|#{@software_filename})
+        %r{(^.*(?:#{@software_name}|#{@software_filename})
           (?:\)|\s\u2122|\s[\d\.]+|\scodec)?\s(?:is\s
           (?:an?|the)|(?:project\s)?provides)[\s\w\'\(\)\,\-\+\/\.\:]+\.(?:\s|$)
-        )/ix
+        )}ix
```

Comment:
Could you carefully review this? I did not find a failing test, but I'm not 100% either that `ix` is respected